src/python/aqueduct_executor/operators/connectors/tabular/load.py

```diff
@@ -7,13 +7,13 @@
 
 class RelationalParams(models.BaseParams):
     table: str
-    update_mode: Optional[common.UpdateMode] = common.UpdateMode.REPLACE
+    update_mode: common.UpdateMode = common.UpdateMode.REPLACE
 
     class Config:
         validate_assignment = True
 
     @validator("update_mode")
-    def set_update_mode(cls, update_mode):
+    def set_update_mode(cls, update_mode):  # type: ignore
```

Comment:
wait why can't we type `update_mode: common.UpdateMode` here? And have a None return value.

Reply:
The validator has some implicit assumption on typing, and mypy complains when the input is `common.UpdateMode`. I think it's fine to ignore the typing check here